tools/ci/maven-utils.sh

```diff
@@ -107,3 +105,6 @@ MVN_GLOBAL_OPTIONS+="--no-snapshot-updates "
 MVN_GLOBAL_OPTIONS+="-B "
 # globally control the build profile details
 MVN_GLOBAL_OPTIONS+="$PROFILE "
```

Comment:
all the options except the `--settings` one should be added to `MVN_GLOBAL_OPTIONS_WITHOUT_MIRROR`, shouldn't they?

Reply:
well this is embarrassing.